File: 110algoritmos/exercicios016/ex016-Go/ex016.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var num float64

	fmt.Println("Digite um numero fracionario: ")
	fmt.Scanln(&num)

	truncado := math.Trunc(num)

	fmt.Printf("\nO numero %v truncado é %v", num, truncado)

}

File: 110algoritmos/exercicios016/ex016-Csharp/ex016.cs
using System;

class Program {
    static void Main(){
        double num, truncado;

        Console.Write("Digite um numero frazionado: ");
        num = double.Parse(Console.ReadLine());

        truncado = Math.Truncate(num);
        Console.Write("Valor: " + num + " truncado é: " + truncado);

        
    }
}